src/bun.js/bindings/wrapAnsi.cpp

```
@@ -0,0 +1,912 @@
+        return 4;
+    return 1; // Invalid, treat as 1
+}
+
+static char32_t decodeUTF8(const Latin1Character* ptr, size_t available, size_t& outLen)
+{
+    uint8_t byte = static_cast<uint8_t>(ptr[0]);
+
+    if (byte < 0x80) {
+        outLen = 1;
+        return byte;
+    }
```

Cppcheck (2.19.0)

[information] Include file

(missingInclude)